Redirect Users to GitHub's Releases Page (#1864)
Cxbx-Reloaded/Cxbx-Reloaded: README.md

    * Make sure to enable winpcap compatibility mode!

## Automated Builds

> Cxbx-Reloaded does not currently have stable builds, but you can go to the Releases tab to obtain pre-release builds or use the links below:

Reviewer: ```suggestion
Cxbx-Reloaded doesn't currently have stable builds, but you can obtain pre-release builds from the Releases tab, or the links below:
```

Author: fixed

  * **[Release Builds](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/releases)**

>     * **WINE users will need to use `CxbxReloaded-Release-VS2017.zip` for it to run.**

Reviewer: You can also word it like this:
```suggestion
    * **WINE users will need to use `CxbxReloaded-Release-VS2017.zip` for it to run correctly.**
```

Author: fixed
